src/core/NestedProportionalAreaChart/ScaledCircle.js

```diff
@@ -35,8 +37,8 @@ function ScaledCircle({
     style: referenceStyle
   } = reference;
   const radii = data.map(d =>
-    d.x !== referenceData.x
-      ? (Math.sqrt(d.x) * size) / Math.sqrt(referenceData.x)
+    d.y !== referenceData.y
+      ? (Math.sqrt(d.y) * size) / Math.sqrt(referenceData.y)
```

Comment:
😱 you mean you were using x this whole time!

Reply:
Damn, we're shooting messengers now? I'm only _fixing_ them :face_with_head_bandage: